src/WebJobs.Extensions.Http/ClaimSlim.cs

```diff
@@ -0,0 +1,50 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Runtime.Serialization;
+using System.Security.Claims;
+
+namespace Microsoft.Azure.WebJobs.Extensions.Http
+{
+    /// <summary>
+    /// Light-weight representation of a <see cref="Claim"/> object.
+    /// </summary>
+    [DataContract]
+    internal struct ClaimSlim
```

Comment:
>ClaimSlim [](start = 20, length = 9)

Add a comment that this needs to be kept in-sync with the Easy Auth version of this same code.